public class Main { 
 public static void main(String[] args) { 
 int a = 10; 
 int b = 0; 
 int result = a / b; 
 System.out.println(result); 
 } 
} 

*/
Q19. What runtime exception is thrown? Why does division by zero cause an issue in Java?
answer->Exception in thread "main" java.lang.ArithmeticException: / by zero at Main.main(Main.java:5).
1.The code will throw a ArithmeticException at runtime because you are trying to divide by zero.
2.In arithmetic operations, dividing by zero is mathematically undefined. This is because any number divided by zero does not yield a finite or meaningful result (i.e., it cannot be represented as a number).
In Java, when you attempt to divide an integer by zero, it leads to a runtime exception (ArithmeticException), which prevents the program from continuing.
3.Why Java Throws an Exception:
Java doesn't produce an "infinity" or "NaN" (Not-a-Number) result for integer division by zero, as it might in floating-point operations.
Instead, Java throws an exception to signal that a serious error has occurred, stopping further execution to avoid unpredictable behavior.
/*
